server-spi/src/main/java/org/keycloak/models/OrganizationModel.java

```diff
@@ -19,10 +19,13 @@
 
 public interface OrganizationModel {
 
+    String USER_ORGANIZATION_ATTRIBUTE = "kc.org";
+
     String getId();
 
     void setName(String name);
 
     String getName();
 
+    boolean addMember(UserModel member);
```

Comment:
I've noticed all other methods which touches managing members in Organization are declared in the `OrganizationProvider`.  What is reasoning behind a decision to have it here and not in the provider? 

I agree it's more convenient to use this method rather than something like `session.organizations().addMemberToOrganization(realm, organization, user)`. On the other hand having the  implementation in single place (`JpaOrganizationProvider`) helps with maintanability, IMO.  

We could probably leverage the same approach what is used already in the codebase: Thereis e.g. this method in `ClientModel`:

https://github.com/keycloak/keycloak/blob/adf838f81ffb84618a1fcac21422cdd188138b21/server-spi/src/main/java/org/keycloak/models/ClientModel.java#L230

and the implementation then forwards the call to the provider: https://github.com/keycloak/keycloak/blob/adf838f81ffb84618a1fcac21422cdd188138b21/model/jpa/src/main/java/org/keycloak/models/jpa/ClientAdapter.java#L355-L357

https://github.com/keycloak/keycloak/blob/adf838f81ffb84618a1fcac21422cdd188138b21/server-spi/src/main/java/org/keycloak/models/ClientProvider.java#L116

and the implementation lives inside a provider implementation.